PlayBooks/Official/Version-1/Large-Company/en/OpenChain Playbook - Large Company.md

```diff
@@ -0,0 +1,192 @@
+2.	 A process or processes to ensure the relevant teams understand the policy.
+3.	 A training program to ensure we can execute the policy.
+
+# We have
+
+1. 	A policy that includes a “permitted list” of projects we are comfortable with and a “denied list” of licenses we are not comfortable with. 
+2.	 Some processes focused on engineering teams using the policy. 
+3.	 A recurring training program for engineers known to be working on open source.
+4.	 A legal department, which may include dedicated staff for compliance in different areas.
+5.	 Requirements for following existing development processes or domain-specific quality standards.
+
+# The People Involved
+Senior engineering, legal and management staff.
+
+## Personas
+### The Senior Engineer
+
+She has been working in this field for ten years and has been with this company for five years. She is experienced in developing proprietary software and has experience in using open source. While not an expert in open source licensing (or general licensing), she has basic knowledge regarding the obligations of key open source licenses and how they impact hardware distribution. This said, she is using “community knowledge” instead of deep personal expertise in the matter.
+
+### The Legal Expert
+
+He has been working in the field of software licensing for eight years and has always worked at this company. He is experienced in other areas of law too, particularly related to trademarks. He has limited experience in open source licensing and has relied on the engineering department to understand market norms. He has spent some time reading open source licenses, interpreting them and seeking to reconcile his interpretation with online resources. The match has not always been perfect.
+
+### The Management Representative
+
+They have been with this company for four years. Previously they were with a competing company for seven years. They have deep experience in product management but have largely relied on members of their teams or departments to deal with the details of software development and legal matters. They are aware that open source process management may need changing their level of involvement or adjustments in previously successful processes.
+
+# Planning Our Strategy
+
+We will use the OpenChain self-certification questionnaire as a way to check our current status versus the status required for OpenChain ISO/IEC 5230:2020 self-certification. This involves understanding the:
+
+1. 	Current policy for managing decisions around open source (and other) software.
+2.	 Current processes being used for managing open source (and other) software.
+3.	 Current records showing the usage of this software over the previous year.
+
+# Executing Our Plan
+
+## Step 1
+
+Identify a key sponsor for the project. Ideally, this will be a senior executive (C-suite) with ability to drive the activities, provide repeated vocal support and assist with budget needs.
+
```

Comment:
"Ideally", but this does not happen in my experience. C-suite is too high level.  It could be someone like a business owner for the initial group doing the first compliance activity.